src/mgmt/pool.pb-c.h

```diff
@@ -54,10 +54,15 @@ struct  _Mgmt__CreatePoolReq
    * DAOS system identifier
    */
   char *sys;
+  /*
+   * Access Control Entries in short string format
+   */
+  size_t n_acl;
```

Comment:
Why size_t?  This would be better as an unsigned int wouldn't it?

Reply:
Can't control this one, unfortunately. It's generated code.